src/main.rs

```diff
@@ -49,7 +49,20 @@ fn main() {
             .arg(Arg::with_name("target").long("target").short("t").required(false)
                  .default_value("fuzzer_script_1")
                  .help("name of the first fuzz target to create")))
-        .subcommand(SubCommand::with_name("run").about("Run the fuzz target in fuzz/fuzzers")
+        .subcommand(SubCommand::with_name("run").long_about("Run the fuzz target in fuzz/fuzzers")
+            .about(
+"
+
+Run the fuzzer on a given target. Example usage:
```

Comment:
It's smooshed up with the version otherwise. Could be one line instead, but I think there should be a line there